src/main/java/service/report/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,22 @@
+package service.report;
+
+import data.db.Storage;
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    private static final String HEADLINE = "fruit,quantity";
+    private static final String SEPARATOR = ",";
+
+    @Override
+    public String getReport() {
+        StringBuilder stringStorage = new StringBuilder();
+        stringStorage.append(HEADLINE);
+
+        for (Map.Entry<String, Integer> storageMap : Storage.fruitsStorage.entrySet()) {
+            stringStorage.append(System.lineSeparator()).append(storageMap.getKey())
+                    .append(SEPARATOR).append(storageMap.getValue());
```

Comment:
The `fruitsStorage` field seems to be accessed directly which violates the encapsulation principle. Consider creating a method in the `Storage` class that can return the necessary data without exposing the internal representation.